keras_cv/layers/preprocessing3d/global_random_dropping_points.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+
+from keras_cv.layers.preprocessing3d import base_augmentation_layer_3d
+
+POINT_CLOUDS = base_augmentation_layer_3d.POINT_CLOUDS
+BOUNDING_BOXES = base_augmentation_layer_3d.BOUNDING_BOXES
+
+
+class GlobalRandomDroppingPoints(base_augmentation_layer_3d.BaseAugmentationLayer3D):
+    """A preprocessing layer which randomly drops point during training.
+
+    This layer will randomly drop points based on keep_probability.
+    During inference time, the output will be identical to input. Call the layer with `training=True` to drop the input points.
+
+    Input shape:
+      point_clouds: 3D (multi frames) float32 Tensor with shape
```

Comment:
I wonder if we need to support 4D tensor [batch_size, num_frames, num_points, point_feat]?
Asking user to do tf.map_fn on their own might be too much cognitive load, WDYT?

Reply:
Yes, it is supported. Please see the test case, where I tested augmentation with a batch dimension.